### PR TITLE
stataUSDC on Base

### PR DESCRIPTION
**Pull request by noisekit**
WIP

### CODE REVIEW

File: liquidity/lib/useApprove/useApprove.ts [hunk 1]
```diff
@@ -88,3 +93,98 @@ export const useApprove = (
...
+  });
+
+  // useApproval returns decimals 18
+  const stataAllowance = allowanceStata && utils.parseUnits(allowanceStata?.toString(), 18);
+  const usdcAllowance = allowanceUSDC && utils.parseUnits(allowanceUSDC?.toString(), 18);
+
+  const amountToApproveUSDC = wei(amount, 6, true).mul(price).toBN();
+  const amountToApproveStata = BigNumber.from(amount);
+
+  // TODO: Multiply USDC amount by stata exchange rate
```
Comment (MrX-SNX) [inline liquidity/lib/useApprove/useApprove.ts:124]: missed a todo?

File: liquidity/lib/useDepositBaseAndromeda/stataReducer.ts
Comment (MrX-SNX) [inline liquidity/lib/useDepositBaseAndromeda/stataReducer.ts:1]: empty file?

File: liquidity/lib/useDepositBaseAndromeda/useDepositBaseAndromeda.tsx [hunk 2]
```diff
@@ -72,10 +73,173 @@ export const useDepositBaseAndromeda = ({
...
+        ](parseUnits(collateralChange.toString(), 6), activeWallet?.address, 0, true);
+
+        const tx = await signer.sendTransaction({
+          ...call,
+          gasLimit: hexlify(500_000),
+        });
+
+        dispatch({ type: 'pending', payload: { txnHash: tx.hash } });
+        await tx.wait();
+        console.log('tx', tx);
```
Comment (MrX-SNX) [inline liquidity/lib/useDepositBaseAndromeda/useDepositBaseAndromeda.tsx:239]: you wanna console log that?

File: liquidity/lib/useDepositBaseAndromeda/useDepositBaseAndromeda.tsx [hunk 3]
```diff
@@ -72,10 +73,173 @@ export const useDepositBaseAndromeda = ({
...
+
+        // Get the logs from the tx
```
Comment (MrX-SNX) [inline liquidity/lib/useDepositBaseAndromeda/useDepositBaseAndromeda.tsx:241]: dont see where you get the logs?

File: liquidity/lib/useApprove/useApprove.ts (hunk 1 above)
Comment (noisekit) [reply liquidity/lib/useApprove/useApprove.ts:124]: it is not my pr, haven't touched the code. I believe this needs to be closed in favour of https://github.com/Synthetixio/liquidity-ui/pull/1 please review that one instead.

**Comment by barrasso** [comment]
Closed in favor of #9

**Pull request closed without merging**